docs/user/ppl/functions/datetime.rst

```diff
@@ -507,13 +506,13 @@ Return type: STRING
 
 Example::
 
-    >od source=people | eval `DATE_FORMAT('1998-01-31 13:14:15.012345', '%T.%f')` = DATE_FORMAT('1998-01-31 13:14:15.012345', '%T.%f'), `DATE_FORMAT(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r')` = DATE_FORMAT(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r') | fields `DATE_FORMAT('1998-01-31 13:14:15.012345', '%T.%f')`, `DATE_FORMAT(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r')`
+    os> source=people | eval `DATE_FORMAT('1998-01-31 13:14:15.012345', '%T.%f')` = DATE_FORMAT('1998-01-31 13:14:15.012345', '%T.%f'), `DATE_FORMAT(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r')` = DATE_FORMAT(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r') | fields `DATE_FORMAT('1998-01-31 13:14:15.012345', '%T.%f')`, `DATE_FORMAT(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r')`
     fetched rows / total rows = 1/1
-    +-----------------------------------------------+----------------------------------------------------------------+
-    | DATE('1998-01-31 13:14:15.012345', '%T.%f')   | DATE(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r')   |
-    |-----------------------------------------------+----------------------------------------------------------------|
-    | '13:14:15.012345'                             | '1998-Jan-31st 01:14:15 PM'                                    |
-    +-----------------------------------------------+----------------------------------------------------------------+
+    +------------------------------------------------------+-----------------------------------------------------------------------+
+    | DATE_FORMAT('1998-01-31 13:14:15.012345', '%T.%f')   | DATE_FORMAT(TIMESTAMP('1998-01-31 13:14:15.012345'), '%Y-%b-%D %r')   |
```

Comment:
I have also changed this in my [PR](https://github.com/Bit-Quill/opensearch-project-sql/pull/204/files#diff-d77a6d15ba9b05d3242ae66e7b487ceebcf8c5fd8ee8eb5c1eb2b9e337d59d7c). Should I revert my changes?

Reply:
No!
That would be interpreted as merge conflict, which is easy to resolve.